cfme/tests/configure/test_analysis_profiles.py

```diff
@@ -24,13 +24,27 @@
     {'Name': files_list[0]['Name'],
      'Collect Contents?': not files_list[0]['Collect Contents?']}]
 
+TENANT_NAME = "tenant_{}".format(fauxfactory.gen_alphanumeric())
 
-def events_check(updates=False):
-    form_bug = BZ(1485953, forced_streams=['5.7', '5.8', 'upstream'])
-    if updates:
-        return updated_files if not form_bug.blocks else None
-    else:
-        return events_list if not form_bug.blocks else None
+OPERATIONS = ["Add", "Edit", "Delete", "Copy"]
+
+PRODUCT_FEATURES_QUOTA = [
```

Comment:
This is much better parametrization than before, thanks!

in-line comments here explaining what this represents would make it even more readable in the future - is this a UI nav path? without a comment, I have to guess and verify my guess.